docs/02 - Contributing/D - Database Migration.md

```diff
@@ -6,41 +6,51 @@
     - [A. Adding a Database Connection](#a-adding-a-database-connection)
     - [B. Generating & Registering a new Changelog File](#b-generating--registering-a-new-changelog-file)
 
-[Liquibase] is used by this project for database migrations. Liquibase is able to generate changelogs (migration scripts) and Spring will automatically apply the changes to the database.
+[Flyway] is used by this project for database migrations. Flyway is able to generate migration scripts (via JPA buddy) 
+and Spring will automatically apply the changes to the database (if Flyway is enabled).
 
 ## Applying Migrations
 
-This project is configured to apply any pending migrations automaticvally when the application is started.
+This project is configured to apply any pending migrations automatically when the application is started. This can be 
+disabled in the Anchor Platform Configuration:
+```text
+spring.flyway.enabled: false
+```
 
 ## Adding a New Migration
 
-We usually generate the migrations using IntelliJ IDEA's JPA Buddy plugin. If you don't have it installed yet, please go ahead and download it at https://plugins.jetbrains.com/plugin/15075-jpa-buddy.
+We usually generate the migrations using IntelliJ IDEA's JPA Buddy plugin. If you don't have it installed yet, please 
+go ahead and download it at https://plugins.jetbrains.com/plugin/15075-jpa-buddy.
 
-There's also an option to generate the migration from command-line or even manually, but we won't be covering that in this tutorial.
+There's also an option to generate the migration from command-line or even manually, but we won't be covering that in 
+this tutorial.
```

Comment:
In a meeting, our partners mentioned they usually do this migration update manually in a separate pipeline. Do you mind covering it in the tutorial, or adding a link to a website that teaches you how to do that?